ClickHelp/SourceJSONConfig.json

```diff
@@ -0,0 +1,54 @@
+{
+    "Services": [
+      {
+        "Url": "{Url of your organization}/api/v1",
```

Comment:
You are right it should be a domain name and not an organization. What does KB stand for? In all cases, clickhelp offers to have a custom domain so really it can be anything.

Reply:
I got the `kb` from the examples on their documentation. 
Thanks for the change, it's clearer now.